priv/blocks/astarte_devices_source.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "astarte_devices_source",
+  "beam_module": "Elixir.Astarte.Flow.Blocks.DeviceEventsProducer",
+  "type": "producer",
+  "schema": {
+    "$id": "https://astarte-platform.org/specs/astarte_flow/blocks/astarte_devices_source.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "AstarteDevicesSourceOptions",
```

Comment:
The `title` property will end up as a user-facing title in an autogenerated form. I know we're following a pattern with the other schemas but I wonder if we could use a more readable title than `AstarteDevicesSourceOptions`

A similar argument goes for the title of each property, which is now missing. For instance, it could be something like:
```
"amqp_routing_key": {
    "type": "string",
    "description": "The routing key used to bind the queue to the exchange.",
    "title": "AMQP routing key"
},
```

Reply:
Updated